server/build.gradle

```diff
@@ -84,11 +84,20 @@ task createVersionFile() {
     def sha = System.getenv("COMMIT_SHORT_SHA")
     if (sha == null) {
         def outputStream = new ByteArrayOutputStream()
-        exec {
-            commandLine "git", "rev-parse", "--short", "HEAD"
-            standardOutput = outputStream
+        try {
```

Comment:
In case if project was downloaded from github as archive and doesn't have .git folder.